pkg/framework/spi/jwe/api.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package jwe
+
+import "github.com/google/tink/go/keyset"
+
+// Provider interface for Crypto ctx
+type Provider interface {
+}
+
+// Creator method to create new pluggable JWE service
+type Creator func(prov Provider) (JWE, error)
+
+// JWE is an interface defining a pluggable JWE builder operations provider to Aries agents.
```

Comment:
How does one call Anoncrypt vs Authcrypt?

Reply:
these would be separate implementations building their unique JWE envelope

Reply:
I suppose we will see how this works in a followup.